compiler/src/passes/instr/decision.rs

```diff
@@ -683,14 +684,21 @@ mod intrinsics {
         };
     }
 
+    macro_rules! of_volatile_funcs {
```

Comment:
The alternative would be a memory category, but I don't have any preference. What ever looks better is fine. You can even have both categories.